torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -3390,6 +3391,732 @@ bool LoopNest::rfactor(
   return true;
 }
 
+namespace randomization_helper {
+
+int64_t max_transformations(int n_max_transforms) {
+  // This env variable helps control the max number of transformations. This can
+  // be helpful in gradually reducing the number of transformations when we see
+  // an error.
+  static const char* enable_c_str =
+      std::getenv("PYTORCH_TENSOREXPR_RANDOM_MAX_TRANSFORM");
+  if (!enable_c_str) {
+    return n_max_transforms;
+  }
+  return std::stoi(std::string(enable_c_str));
+}
```

Comment:
Can we use existing `PYTORCH_JIT_OPT_LIMIT` mechanism instead? It kinda does the same thing.

https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/OVERVIEW.md#jit-optimization-limiter

Reply:
Thanks. Moved the random loop nest transformations to a new file so that I can set up the env variable correctly. This also helps in better separation of code.

Reply:
Added `setUp` and `tearDown` functions, so it should alter the environment only during the specific test.